### PR TITLE
docs(npx-version): mentioned using a renovate regex manager to update the version

### PR DESCRIPTION
**Pull request by travi**
confirmed in https://github.com/travi-test/semantic-release-tester/commit/0c14764598290d9869fa1a0f6e4d3b1885d23ee5

### CODE REVIEW

File: docs/usage/installation.md
Comment (HonkingGoose) [inline docs/usage/installation.md:33]:
```suggestion
   You can use [Renovate's regex manager](https://docs.renovatebot.com/modules/manager/regex/) to get updates for **semantic-release**.
   Put this in your Renovate configuration file:
```

File: docs/usage/installation.md
Comment (HonkingGoose) [inline docs/usage/installation.md:47]:
```suggestion
3. `npx` is a tool bundled with `npm@>=5.2.0`. You can use it to install (and run) the semantic-release binary.
```

**Review by HonkingGoose — COMMENTED** [review]
My goals with this rewrite are to:

- Clarify _who_ is recommending things here
- Explain that pinning to a specific version of **semantic-release** means users (or a bot) needs to check for updates often
- Use simple words and phrases
- Use `you` to put the reader into the text
- Use `must` to indicate requirements

File: docs/usage/installation.md
Comment (travi) [reply docs/usage/installation.md:31]: i have not tested the regex manager config suggested in this PR against an npx command that specifies only the major version. it is unclear if it would update only the major version, replace the partial definition with a full one, or fail in some way. is there a difference in this case that is worth clarifying in some way, or should both scenarios be expected to work with the suggested config?

**Comment by travi** [comment]
thanks for taking the time to provide feedback, @HonkingGoose! much of this change was just moving existing content into a more appropriate structure, but your suggestions do improve the content, so much appreciated.

File: docs/usage/installation.md
Comment (HonkingGoose) [inline docs/usage/installation.md:43]: Is this matchStrings valid for both _pinned_ and _normal_ versions? Or just for one of those?

Comment (travi) [reply docs/usage/installation.md:43]: good question. i wondered the similar. i tested with:
* [just the major version](https://github.com/travi-test/semantic-release-tester/pull/418/files)
* [a fully pinned version](https://github.com/travi-test/semantic-release-tester/pull/417/files)

both were recognized, since the expression ends with the whitespace after the version, and provided updates as i would have expected. i think this covers the scenarios suggested in this portion of the docs.

out of curiosity, i've also tried [specifying the version as simply `beta`](https://github.com/travi-test/semantic-release-tester/commit/d359f6051719a644270cae7542be59532702f06f) since i wasnt sure how that would be handled with no additional configuration. i did not receive an update PR, which is good since i normally use that when testing a pre-release channel while working through a series of updates. i'm still curious what would happen if i specified a precise pre-release version, like `semantic-release@22.0.0-beta.1`, but have not tried that out yet. i think that curiosity goes beyond the goals of this PR, though, so i can experiment further later.

Comment (HonkingGoose) [reply docs/usage/installation.md:43]: > both were recognized, since the expression ends with the whitespace after the version, and provided updates as i would have expected. i think this covers the scenarios suggested in this portion of the docs.

Can you explain in the docs what works/doesn't work? That way readers don't have to guess.

> i'm still curious what would happen if i specified a precise pre-release version, like `semantic-release@22.0.0-beta.1`, but have not tried that out yet. i think that curiosity goes beyond the goals of this PR, though, so i can experiment further later.

I don't know if the `matchStrings` "version regex" is "wide" enough to include `-beta` versions. I don't know enough RegEx to check this. I think you should try it, and see what happens. 😄 

Also if you made it "wide" enough, so it captures `-beta` you should then find a way to avoid the `--` CLI flags.

**Review by gr2m — APPROVED** [review]
nice!

File: docs/usage/installation.md
Comment (travi) [reply docs/usage/installation.md:43]: > Can you explain in the docs what works/doesn't work? That way readers don't have to guess.

i decided to split the mention of the regex manager to a separate note so that i could clarify that it was an automation option for either pinning strategy. i think that provides better clarity for those scenarios

> I don't know if the matchStrings "version regex" is "wide" enough to include -beta versions. I don't know enough RegEx to check this. I think you should try it, and see what happens. 😄

the `matchStrings` regex does match versions that include the pre-release version. i think those scenarios are beyond the scope of what these particular docs need to cover, so i'm comfortable investigating those details after getting these updates merged.

> Also if you made it "wide" enough, so it captures -beta you should then find a way to avoid the -- CLI flags.

since the regex only matches up to the next whitespace character, other trailing flags would already be excluded

Comment (HonkingGoose) [reply docs/usage/installation.md:43]: Thanks for responding to me again `@travi`, and for improving the docs again.

I'll let you resolve this comment. 😉 I don't have the rights to do that, for whatever reason.

File: docs/usage/installation.md
Comment (HonkingGoose) [inline docs/usage/installation.md:50]: I think **semantic-release** needs to be bold:

```suggestion
4. `npx` is a tool bundled with `npm@>=5.2.0`. You can use it to install (and run) the **semantic-release** binary.
```

**Review by HonkingGoose — APPROVED** [review]

File: docs/usage/installation.md
Comment (travi) [reply docs/usage/installation.md]: good catch

**Pull request merged**
